### DISCUSSION TITLE
Why can't I see my published Action in the Marketplace?

### DISCUSSION THREAD

**Discussion by aormsby** [community/community#26083] [Actions]
<p>I’ve published an Action, and I’ve seen the page for it. However, I can’t find it if I search in the Marketplace. Why wouldn’t it show up for me? Is it just possible that we can’t search for our own released work? That seems weird.</p>
<p>I just want to verify that it’s showing up as expected. Can someone try to search for it?</p>
<p>You should be able to find it by name - ‘Fork Sync With Upstream’. Here’s the active page link for reference: <a href="https://github.com/marketplace/actions/fork-sync-with-upstream" rel="nofollow noopener">https://github.com/marketplace/actions/fork-sync-with-upstream</a></p>

**Comment by andymckay** [comment, marked as answer]
<p>Good question! It should be showing up. I’ll file an issue with the appropriate engineering team and ask. Sorry about that.</p>

**Comment by aormsby** [comment]
<p>Sweet. I figured that wasn’t quite right. Thanks. :slight_smile:</p>

**Comment by andymckay** [comment]
<p>This should be showing up now.</p>

**Comment by aormsby** [comment]
<p>Fix confirmed! Much appreciated!</p>
